packages/eslint-plugin/src/rules/no-deprecated.ts

```diff
@@ -116,17 +116,21 @@ export default createRule({
     function getJsDocDeprecation(
       symbol: ts.Signature | ts.Symbol | undefined,
     ): string | undefined {
-      const tag = symbol
-        ?.getJsDocTags(checker)
-        .find(tag => tag.name === 'deprecated');
+      try {
+        const tag = symbol
+          ?.getJsDocTags(checker)
+          .find(tag => tag.name === 'deprecated');
 
-      if (!tag) {
-        return undefined;
-      }
+        if (!tag) {
+          return undefined;
+        }
 
-      const displayParts = tag.text;
+        const displayParts = tag.text;
 
-      return displayParts ? ts.displayPartsToString(displayParts) : '';
+        return displayParts ? ts.displayPartsToString(displayParts) : '';
+      } catch {
+        return;
```

Comment:
could you add a comment that says something like `// workaround for https://github.com/microsoft/TypeScript/issues/60024`?